content/master/whats-crossplane/_index.md

```diff
@@ -223,6 +224,62 @@ GCP, Terraform, Helm, GitHub, etc to support Crossplane v2 soon.
 <!-- vale gitlab.FutureTense = YES -->
 {{</hint>}}
 
+### Operations
+
+Operations let you run operational tasks using function pipelines.
+
+While composition and managed resources focus on creating and managing
+infrastructure, operations handle tasks that don't fit the typical resource
+creation pattern - like certificate monitoring, rolling upgrades, or scheduled
+maintenance.
+
+**Operations run function pipelines to completion like a Kubernetes Job.**
+Instead of continuously managing resources, they perform specific tasks and
+report the results.
+
+```mermaid
```

Comment:
this is a nice diagram, but we don't seem to give any context on what it does at a high level - should we?

Reply:
Good point - done.